pkg/cmd/get_clustercompliancereport.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/aquasecurity/starboard/pkg/apis/aquasecurity/v1alpha1"
+	"github.com/aquasecurity/starboard/pkg/compliance"
+	"github.com/aquasecurity/starboard/pkg/starboard"
+	"github.com/spf13/cobra"
+	"io"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strconv"
+)
+
+func NewGetClusterComplianceReportsCmd(executable string, cf *genericclioptions.ConfigFlags, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "clustercompliancereports (NAME)",
+		Aliases: []string{"clustercompliance"},
+		Short:   "Get cluster compliance reports",
+		Long:    `Get cluster compliance report for pre-define spec`,
+		Example: fmt.Sprintf(`
+# Get cluster compliance report for specifc spec in JSON output format
+  %[1]s get clustercompliancereports nsa -o json
+
+# Get compliance detail report for control checks failure in JSON output format
+  %[1]s get clustercompliancereports nsa -o json --detail`, executable),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			//init client dependencies
```

Comment:
Here and in other places: Do we need to describe the code that we write? IMO, it's pretty obvious that we use client-go to construct a new clientset just by looking at Go code.